### DISCUSSION TITLE
@font-face

### DISCUSSION THREAD

**Discussion by kondratiukD** [community/community#122452] [Programming Help]
### Select Topic Area

Question

### Body

I have a problem with @font-face. I use several font font-family on my website. And one of them has different styles for H1 and H2 but the same font-weight: 700;. I connected them all but the font on H2 does not want to be displayed. Other fonts are displayed but the one I need is not.  
 
@font-face {
  font-display: swap;
  font-family: 'Play1';
  font-style: normal;
  font-weight: 700;
  src: url('../fonts/play-v19-latin-700.woff2') format('woff2');
}

@font-face {
  font-display: swap;
  font-family: 'Play2';
  font-style: normal;
  font-weight: 700;
  src: url('../fonts/play-v19-latin-regular.woff2') format('woff2'); 
} 

@font-face {
  font-display: swap;
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 600;
  src: url('../fonts/roboto-condensed-v27-latin-600.woff2') format('woff2');
}

@font-face {
  font-display: swap;
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url('../fonts/roboto-v30-latin-regular.woff2') format('woff2');
}   

Here's how I highlight it for each heading because the same weight -

h1 {
  font-family: 'Play1', sans-serif;
}

h2 {
  font-family: 'Play2', sans-serif;
}

My body -

body {
  color: var(--color-brand-dark);
  font-family: 'Roboto', 'Play1', 'Play2', sans-serif;
  font-size: 18px;
  font-weight: 400;
  line-height: 1.2;
  margin: 0;
  background-color: var(--color-primare-light);
  overflow-x: hidden;
}

All fonts are connected locally  -
<img width="179" alt="Знімок екрана 2024-05-05 141816b" src="https://github.com/community/community/assets/158787270/9b794c33-2732-401b-8c0e-9fa7613e5edc">

Other fonts work, but not this one. I don't know what to do anymore, I hope you can help me

**Comment by aroonmircam** [comment]
Some suggestions that you can apply

1. Check the font format: Make sure the font file format (woff2) is compatible with all browsers you are using. If necessary, provide additional formats such as woff or ttf for compatibility.

2. Clear the browser cache: Sometimes font display problems can be caused by the browser cache. Try clearing the browser cache or loading your site in an incognito window to see if that solves the problem.

3. Check the CSS cascading rules: Make sure there are no other CSS rules that are overriding the font settings for h2 elements. You can use the browser's developer tools to inspect the styles applied to your h2 elements and check for conflicts.

4. Try different font-weight values: Although you are using the same weight (700) for both fonts, there may be differences in how they render in different browsers or operating systems. Try experimenting with different font-weight values to see if that solves the problem.
